torch/distributed/_tensor/dispatch.py

```diff
@@ -148,15 +150,111 @@ def operator_dispatch(
     suggested_input_schema = output_sharding.schema_suggestions[0]
     needs_redistribute = suggested_input_schema is not op_schema
 
+    def _arg_type_is_primitive(arg_type):
+        _python_primitive_types = [
+            "bool",
+            "int",
+            "float",
+            "double",
```

Comment:
there's no "double" in the native_functions schema, `float` is the only type.

Reply:
this seems not getting addressed.

Reply:
I haven't updated the PR to github but I have removed "double' and also the "(type == "float") or (type == "double")" if-branch because we don't have aten op that hits the dispatcher and also returns a scalar value.